components/security/secure_tools.rst

```diff
@@ -9,39 +9,28 @@ Generating a Secure random Number
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Whenever you need to generate a secure random number, you are highly
-encouraged to use the Symfony
-:class:`Symfony\\Component\\Security\\Core\\Util\\SecureRandom` class::
+encouraged to use the 
+:phpfunction:`random_bytes` function::
 
-    use Symfony\Component\Security\Core\Util\SecureRandom;
+    $random = random_bytes(10);
 
-    $generator = new SecureRandom();
-    $random = $generator->nextBytes(10);
-
-The
-:method:`Symfony\\Component\\Security\\Core\\Util\\SecureRandom::nextBytes`
-method returns a random string composed of the number of characters passed as
-an argument (10 in the above example).
-
-The SecureRandom class works better when OpenSSL is installed. But when it's
-not available, it falls back to an internal algorithm, which needs a seed file
-to work correctly. Just pass a file name to enable it::
-
-    use Symfony\Component\Security\Core\Util\SecureRandom;
-
-    $generator = new SecureRandom('/some/path/to/store/the/seed.txt');
-
-    $random = $generator->nextBytes(10);
-    $hashedRandom = md5($random); // see tip below
+The function returns a random string, suitable for cryptographic use, of
+the number bytes passed as an argument (10 in the above example).
 
 .. note::
 
     If you're using the Symfony Framework, you can get a secure random number
     generator via the ``security.secure_random`` service.
 
+.. note::
+
+    PHP versions 7.0.0 and up provide the ``random_bytes()`` function natively,
```

Comment:
Would write:

``PHP 7 and up .... natively, for older versions we provide a polyfill.``

polyfill should be a link to more information IMO